model-engine/model_engine_server/inference/vllm/vllm_server.py

```diff
@@ -287,11 +287,18 @@ def parse_args():
         model_config,
         served_model_names,
         args.response_role,
-        args.lora_modules,
-        args.chat_template,
+        lora_modules=args.lora_modules,
+        chat_template=args.chat_template,
+        prompt_adapters=args.prompt_adapters,
```

Comment:
will we need to add these to batch?

Reply:
yeah we probably need to, will do this

Reply:
actually I think we don't need to make changes to batch, doesn't look like it asks for anything OAI related